### PR TITLE
Expand documentation on k8s logging

### PR DESCRIPTION
**Pull request by iuriaranda**
- Adds some caveats on ES indexes
- Merges kibana documentation and removes duplicate information

As per https://github.com/skyscrapers/engineering/issues/211

### CODE REVIEW

File: kubernetes/logging.md [hunk 1]
```diff
@@ -144,8 +144,20 @@ As you can see, the original JSON map is in the `_source.log_json` attribute of
 
 ## Log browsing and filtering
 
+To login into your cluster Kibana you should head to https://kibana.<<clustername>>. The login is managed like our other tools with Dex.
```
Comment (venturel) [inline kubernetes/logging.md]: Kibana cluster

Comment (duboisph) [reply kubernetes/logging.md]: also the `<< >>` doesn't render correctly

Comment (iuriaranda) [reply kubernetes/logging.md]: actually no, I want to say something like "Kibana in your cluster" 😅 let me see if I can rewrite it so it's more clear

Comment (iuriaranda) [reply kubernetes/logging.md]: actually, the login info was already mentioned at the beginning of the document, so I've removed this line.

**Pull request merged**